pyproject.toml

```diff
@@ -0,0 +1,51 @@
+[build-system]
+requires = [
+  "setuptools >= 64",
+  "setuptools_scm >= 8",
+]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "requests-unixsocket"
+description = "Use requests to talk HTTP via a UNIX domain socket"
+requires-python = ">= 3.9"
+dependencies = [
+  "requests >= 1.1",
+]
+classifiers = [
+  "Development Status :: 3 - Alpha",
+
+  "Intended Audience :: Developers",
+  "Intended Audience :: Information Technology",
+
+  "Operating System :: OS Independent",
+
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3 :: Only",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: 3.12",
+  "Programming Language :: Python :: 3.13",
+  "Programming Language :: Python :: Implementation",
+  "Programming Language :: Python :: Implementation :: CPython",
+  "Programming Language :: Python :: Implementation :: Jython",
+  "Programming Language :: Python :: Implementation :: PyPy",
+]
+dynamic = [
+  "version",
+]
+
+[[project.authors]]
```

Comment:
FYI, there's also `[[project.maintainers]]` that can be filled with multiple entries as well.